### PR TITLE
CR1 for Mars blog

### PR DESCRIPTION
**Pull request by alexeykazakov**
**Do not publish yet!**

### CODE REVIEW

**Comment by alexeykazakov** [comment]
@maxandersen please review.

File: blog/cr1-for-mars.adoc [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+== What is new ? 
+
+The full details of what is new is available on http://tools.jboss.org/documentation/whatsnew/jbosstools/4.3.0.CR1.html[this page]. Some highlights are below.
+
+== Eclipse Mars.1
+
+This version of JBoss Tools targets Eclipse Mars.1 which includes many bug fixes and some improvements.
+
+=== WildFly 10 and EAP 7 Server Adapters
```
Comment (maxandersen) [inline blog/cr1-for-mars.adoc]: would add the livereload Cmd+3 additon.

Comment (alexeykazakov) [inline blog/cr1-for-mars.adoc]: It's here. See below.

File: blog/cr1-for-mars.adoc [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+New server adapters for JBoss EAP 7 and WildFly 10 have been added to the toolset, allowing you to enjoy all the past benefits, but with all the newest runtimes.
+
+image::images/cr1-for-mars/wildfly10.png[width="500"]
+
+=== Docker tooling
+
```
Comment (maxandersen) [inline blog/cr1-for-mars.adoc]: Add a segment like:

Note: The Docker tooling is part of Eclipse Mars.1 We worked on these improvements to make them rock in JBoss Tools with OpenShift support - so we wanted to highlight them.

Comment (maxandersen) [inline blog/cr1-for-mars.adoc]: Maybe just take the Eclipse Mars.1 section and put before the docker and have it as the last part of the blog ? then all our jboss tools specific stuff comes first and the Mars.1 specific highlights are toghether.

**Comment by nickboldt** [comment]
Should there be mention of JDK8 requirement and installation requirements? I didn't see a link to any downloads. Compare with Beta2 blog... https://github.com/alexeykazakov/jbosstools-website/blob/CR1ForMarsBlog/blog/beta2-for-mars.adoc

Also, should we mention the IS stuff is coming soon?

**Comment by alexeykazakov** [comment]
1. The JDK8 requirement is not new.
We highlighted it in the Beta1 blog:
http://tools.jboss.org/blog/2015-06-23-beta1-for-mars.html#java-8-to-run-eclipse-older-runtimes-ok-for-builds-deployment
2. Yes, I will add a link to the JBT and JBDS downloads. But don't think it's so important to add an installation section in every blog post for every milestone release though.
3. Will add the note about IS too.

**Comment by alexeykazakov** [comment]
Rebased and pushed to master and production.

**Pull request closed without merging**